bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -164,10 +171,15 @@ num_mc_injs = opts.num_mc_injections
 np.random.seed(opts.seed)
 logging.info("Setting random seed to %d.", opts.seed)
 
-# Set output directory
-logging.info("Setting output directory.")
-if not os.path.isdir(outdir):
-    os.makedirs(outdir)
+# Set output directories
+outdir = None
+for output_file in [opts.exclusion_dist_output_file,
+    opts.background_output_file, opts.onsource_output_file]:
+    if output_file is not None:
+        logging.info("Setting output directory.")
+        outdir = os.path.split(os.path.abspath(output_file))[0]
+        if not os.path.isdir(outdir):
+            os.makedirs(outdir)
```

Comment:
I would add a `break` at the end to avoid repeating logging messages.

Reply:
Would it make more sense to move the message inside the last if and specify the directory being created?

Reply:
I agree. Maybe we can also add the directroy name to the log, for a more complete output.